test/cmd/diagnostics.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/../..
+source "${OS_ROOT}/hack/util.sh"
+source "${OS_ROOT}/hack/cmd_util.sh"
+os::log::install_errexit
+
+# This test validates the diagnostics command
+
+# available diagnostics (2015-12-21):
+# AnalyzeLogs ClusterRegistry ClusterRoleBindings ClusterRoles ClusterRouter ConfigContexts DiagnosticPod MasterConfigCheck MasterNode NodeConfigCheck NodeDefinitions UnitStatus
+# Without things feeding into systemd, AnalyzeLogs and UnitStatus are irrelevant.
+# ConfigContexts runs into some invalid contexts created by other tests, so is omitted.
```

Comment:
I ran the test, and it's actually users without auth that are being left lying around, specifically.
```
client config context '/127-0-0-1:28443/cascaded-user'
config context '/127-0-0-1:28443/orphaned-user'
```
These are created in [admin.sh](https://github.com/openshift/origin/blob/master/test/cmd/admin.sh#L21-L22) where the users are deleted but not the kubeconfig entries. I guess I could just have it also clean up the config.